src/styles/MessageList.scss

```diff
@@ -49,6 +49,7 @@
   padding: var(--xs-p);
   font-family: var(--second-font);
   text-align: center;
+  margin-top: 5px;
```

Comment:
Hard-coding 5px seems a bit weird, but I don't have the full picture.

Reply:
You're right, I didn't notice we had variable for that, will fix this